x-pack/plugin/esql/src/internalClusterTest/java/org/elasticsearch/xpack/esql/action/CrossClustersEnrichIT.java

```diff
@@ -61,6 +62,7 @@
 import static org.hamcrest.Matchers.equalTo;
 import static org.hamcrest.Matchers.greaterThanOrEqualTo;
 
+@TestLogging(value = "org.elasticsearch.xpack.esql:DEBUG", reason = "finding bugs due to inconsistent plans")
```

Comment:
NIT: I believe `TestIssueLogging` is dedicated for logging configuration for issue troubleshooting while `TestLogging` is mento for configuring logging for test cases (such as testing a warning/info/debug is logged in certain case)